jenkins/main.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+main() {
+    # ignore trigger workaround for atlassian bug
+    # https://jira.atlassian.com/browse/JRASERVER-64216
+    if [ ${TRIGGER} = "issue_commented" ] \
+        || [ ${TRIGGER} = "issue_comment_edited" ] \
+        || [ ${TRIGGER} = "issue_comment_deleted" ]; then return 0 ; fi
+
+    # remote repo where submissions are pushed for review
+    REPO_URL="github.com/${GITHUB_USERNAME}/workspace-microapps-test-bundles.git"
+    REPO_NAME="workspace-microapps-test-bundles"
+
+    # clone workspace microapps bundles repo and enter
+    git clone "https://${REPO_URL}" --branch master
+    cd $REPO_NAME
+
+    # create new or checkout existing branch
+    git fetch origin -a
+    if git branch -a | grep $JIRA_ISSUEID ; then
+        git checkout --track origin/$JIRA_ISSUEID
+    else
+        git checkout -b $JIRA_ISSUEID
+    fi
+
+    # manipulate the files via python script
+    python ../format-submission.py \
+    --svcacctName "${SVCACCT_NAME}" \
+    --svcacctPwd "${SVCACCT_PWD}" \
+    --issueId "${JIRA_ISSUEID}" || return 0
+
+    # add and push files to remote branch
+    git add *
```

Comment:
It would be nicer here to make it more explicit like `git add *.mapp` or similar just to avoid any accidents